### DISCUSSION TITLE
How to export ODF diagram after importing from VPSC

### DISCUSSION THREAD

**Discussion by ddyyasfh** [mtex-toolbox/mtex#1793] [Ask Anything]
Dear All,
 I followed the tutorial to get out of the polar chart smoothly，But I had trouble coming out with the ODF diagram.May I ask if any one could help with this issue? Many thanks.
[https://mtex-toolbox.github.io/VPSCImport.html](url)

**Comment by ralfHielscher** [comment]
Can you please explain a little bit more in detail what you want to do?

Ralf.
